### ISSUE TITLE
Can't install a package

### ISSUE THREAD

**Issue by AnastasiaDunbar** [wbond/package_control#1271]
Had troubles with bz2 and missing dependencies. Removed Package Control files and reinstalled.
```
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.quick_panel
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
plugins loaded
Package Control: Not running package cleanup since bootstrapping is not yet complete
Exception in thread Thread-1:
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "./threading.py", line 858, in run
  File "2_bootstrap in C:\Users\Admin\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 123, in _background_bootstrap
  File "package_control.loader in C:\Users\Admin\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 207, in add_or_update
  File "package_control.loader in C:\Users\Admin\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 291, in add
  File "./zipfile.py", line 919, in __init__
PermissionError: [Errno 13] Permission denied: '0_package_control_loader.sublime-package'
```

Trying to install a package:

```
Traceback (most recent call last):
  File "package_control.package_installer in C:\Users\Admin\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 154, in on_done
  File "package_control.package_disabler in C:\Users\Admin\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 76, in disable_packages
ImportError: No module named 'package_control'
```

Package control is not in ignored packages.

Error messages:
![errors](https://user-images.githubusercontent.com/11806048/30736800-c8e64dd2-9fb6-11e7-89e4-5338faaca57b.png)

This happened because I removed Monokai Pro while I had the theme on.

**Comment by craigcolesspindrift** [comment]
having same issue, haven't removed anything just trying to install pug-lint
`Exception in thread Thread-9:
Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner`

**Comment by favasconcelos** [comment]
@craigcolesspindrift maybe yours is the same as #1267.

**Comment by bmcminn** [comment]
Just had a similar issue where Package Control was installed, but couldn't use the commands in the command palette.

On a whim, I checked my preferences and discovered Package Control as listed in my `ignored_packages`. Once I deleted that reference, it reloaded and works just fine.

You might double check this and see what's up. Not sure why it auto populated in there, but worth a look I think.

**Comment by AnastasiaDunbar** [comment]
I checked but no.

**Comment by craigcolesspindrift** [comment]
@favasconcelos it was! thanks, for anyone else, I had to remove the open vpn cert and it worked immediatley

**Comment by wbond** [comment]
Users with the charmap issue should go to https://github.com/wbond/package_control/issues/1293#issuecomment-342595937.

**Comment by TimeDelta** [comment]
I'm also having this issue. I am on ST3 build 3103. I even tried using the latest prerelease at package.io/prerelease/Package%20Control.sublime-package but still get the same error. Here are the full contents of the console:

```DPI scale: 1.25
startup, version: 3103 windows x32 channel: stable
executable: /C/Program Files (x86)/Sublime Text 3/sublime_text.exe
working dir: /C/Program Files (x86)/Sublime Text 3
packages path: /C/Users/PL67830/AppData/Roaming/Sublime Text 3/Packages
state path: /C/Users/PL67830/AppData/Roaming/Sublime Text 3/Local
zip path: /C/Program Files (x86)/Sublime Text 3/Packages
zip path: /C/Users/PL67830/AppData/Roaming/Sublime Text 3/Installed Packages
ignored_packages: ["Vintage"]
pre session restore time: 0.340885
startup time: 0.394885
first paint time: 0.406885
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.profile_syntax_definition
reloading plugin Default.quick_panel
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin Python.syntax_test_python
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
plugins loaded
Package Control: Not running package cleanup since bootstrapping is not yet complete
Exception in thread Thread-1:
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "./threading.py", line 858, in run
  File "2_bootstrap in C:\Users\PL67830\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 123, in _background_bootstrap
  File "package_control.loader in C:\Users\PL67830\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 207, in add_or_update
  File "package_control.loader in C:\Users\PL67830\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 291, in add
  File "./zipfile.py", line 919, in __init__
PermissionError: [Errno 13] Permission denied: '0_package_control_loader.sublime-package'

Package Control: Fetching list of available packages and dependencies
  Platform: windows-x32
  Sublime Text Version: 3103
  Package Control Version: 3.3.1-beta4
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 30
  Resolved IP: 50.116.34.243
Package Control: WinINet Debug Proxy
  proxy: 
  proxy bypass: 
  proxy username: 
  proxy password: 
Package Control: WinINet HTTPS Debug General
  Server SSL Certificate:
    subject: packagecontrol.io
    issuer: US, GeoTrust Inc., RapidSSL SHA256 CA
    common name: packagecontrol.io
    issue date: Sat, 26 Nov 2016 19:00:00 GMT
    expire date: Wed, 27 Nov 2019 18:59:59 GMT
Package Control: WinINet HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  User-Agent: Package Control v3.3.1-beta4
  Host: packagecontrol.io
  Connection: Keep-Alive
  Cache-Control: no-cache
  If-None-Match: W/"5aabe7ae-31a100"
  Accept-Encoding: gzip,deflate
  If-Modified-Since: Fri, 16 Mar 2018 15:50:06 GMT
Package Control: WinINet HTTPS Debug Read
  HTTP/1.1 200 OK
  Server: nginx
  Date: Fri, 16 Mar 2018 16:06:40 GMT
  Content-Type: application/json
  Last-Modified: Fri, 16 Mar 2018 16:00:06 GMT
  Transfer-Encoding: chunked
  Connection: keep-alive
  Vary: Accept-Encoding
  ETag: W/"5aabea06-31a100"
  Strict-Transport-Security: max-age=15552000
  Content-Encoding: gzip
Package Control: Caching https://packagecontrol.io/channel_v3.json in C:\Users\PL67830\AppData\Roaming\Sublime Text 3\Packages\User\Package Control.cache\01524fae79697630d0454ba3fabd9414
Traceback (most recent call last):
  File "package_control.package_installer in C:\Users\PL67830\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 154, in on_done
  File "package_control.package_disabler in C:\Users\PL67830\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 76, in disable_packages
ImportError: No module named 'package_control'
Package Control: WinINet HTTPS Debug General
  Closing connection to packagecontrol.io on port 443 after 1 request```

**Comment by Pictor13** [comment]
Ensure also *"0_package_control_loader"* is not in the `ignored_packages`. That solved my headache.
